rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -486,7 +491,7 @@ void MasterGraph::output_routine()
             if (_meta_data_reader)
                 _meta_data_reader->lookup(this_cycle_names);
 
-            full_batch_image_names += this_cycle_names;
+            full_batch_image_names = this_cycle_names;
```

Comment:
please remove this_cycle_names and intialize directly to image_names_batch (rename full_batch...)

Reply:
Done